spec/support/validates_response.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.configure do |_config|
+  def validate_response(response, schema: nil, version: 'v1', strict: true, status: 200)
```

Comment:
```suggestion
  def validate_response(response, schema: nil, version: 'v2', strict: true, status: 200)
```

Reply:
Just to be clear, all existing apis are unversioned and you get the same api behaviour whether you specify v1/v2. Given the frontend/suppliers are integrating with v2 I think it makes sense to have v2 as the default.

Reply:
default now v2 👍